src/Sylius/Bundle/LocaleBundle/Twig/LocaleExtension.php

```diff
@@ -37,12 +39,51 @@ public function __construct(LocaleHelper $helper)
     public function getFilters()
     {
         return array(
+            new \Twig_SimpleFilter('sylius_localized_date', array($this, 'filterLocalizedDate'),
+                array('needs_environment' => true)),
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return array(
             new \Twig_SimpleFunction('sylius_locale', array($this, 'getCurrentLocale')),
+            new \Twig_SimpleFunction('sylius_language', array($this, 'getCurrentLanguage')),
+            new \Twig_SimpleFunction('sylius_direction', array($this, 'getCurrentDirection')),
+            new \Twig_SimpleFunction('sylius_calendar', array($this, 'getCurrentCalendar')),
+            new \Twig_SimpleFunction('sylius_rtl', array($this, 'isRtl')),
         );
     }
 
     /**
-     * @return string
+     * Returns localized language name from context or passed locale code.
+     *
+     * @param string|null $code Locale code to translate current language in.
+     *
+     * @return string Localized language name.
+     */
+    public function getCurrentLanguage($code = null)
```

Comment:
It returns locale name, not language name. It is usually language name combined with country name, like for `en_GB` it is `English (Great Britain)`.